slices/dpkg.yaml

```diff
@@ -6,7 +6,9 @@ essential:
 slices:
   bins:
     essential:
+      - diffutils_bins  # used by dpkg -i
```

Comment:
from what i can tell it's just `cmp` (not all of `diffutils`) and only used by `dpkg-db-backup`. i'd subslice it.